__tests__/cmds/openapi/index.test.ts

```diff
@@ -1350,37 +1330,24 @@ describe('rdme openapi', () => {
     });
 
     it('should send proper headers in GitHub Actions CI for spec hosted at URL', async () => {
-      expect.assertions(8);
       const registryUUID = getRandomRegistryId();
       const spec = 'https://example.com/openapi.json';
 
-      // TODO: move all of this boilerplate to the top-level once we migrate everything over to MSW
-      const server = setupServer(...[]);
+      const mock = getAPIMock()
+        .post('/api/v1/api-registry', body => body.match('form-data; name="spec"'))
+        .reply(201, { registryUUID });
 
-      server.listen({ onUnhandledRequest: 'error' });
+      const exampleMock = nock('https://example.com').get('/openapi.json').reply(200, petstoreWeird);
 
-      server.use(
```

Comment:
msw i think you're really neat but my dude... the code bloat

Reply:
yeah... once nock started using the same underlying interceptor, it was game over